straxen/plugins/daqreader.py

```diff
@@ -156,6 +156,18 @@ def _load_chunk(self, path, start, end, kind='central'):
         first_start, last_start, last_end = None, None, None
         if len(records):
             first_start, last_start = records[0]['time'], records[-1]['time']
+
+            if last_start == end:
```

Comment:
Maybe do it only when a tracked option == ``True`` in order to change the lineage for now?